### PR TITLE
Track II code  + naive featurelization code

### PR DESCRIPTION
**Pull request by LuxiWang99**
Both the track II code from 2 weeks ago and the new naive featurelizaiton code are in the classifier.ipynb. 
The featurelization code generates an edgelist with all edges represented as integers. It also generates a mapping file, in which the order corresponds to the integer they map to. 
The edgelist "result.edgelist" should be ready to be processed by node2vec (c++ version).

### CODE REVIEW

File: data/graphs/backdoor/31a6a74c272af95f5223602e12ef1364527f584675445530eee060e635d2cafb.apk.top.dot [hunk 1]
```diff
@@ -1,37 +0,0 @@
-strict digraph  {
```
Comment (izgzhen) [inline data/graphs/backdoor/31a6a74c272af95f5223602e12ef1364527f584675445530eee060e635d2cafb.apk.top.dot]: Why is this file deleted?

**Comment by izgzhen** [comment]
I found that there is code duplication between the nodebook and python code in `code`.

Can you:

1. Keep the comparison between three types of classifiers inside the nodebook but not in Python, and move `classifier.ipynb` to `code` as well
2. Use `.py` code for the edge list generation, and remove the same code in notebook

This is because ideally the featurization tool would be a python library, while comparative study is best presented as a python notebook.

File: classifier.ipynb [hunk 2]
```diff
@@ -39,6 +40,86 @@
...
+   "outputs": [],
+   "source": [
+    "# random forest\n",
+    "from sklearn.ensemble import RandomForestClassifier\n",
+    "from sklearn.model_selection import cross_val_score\n",
+    "clf = RandomForestClassifier(n_estimators=200, max_depth=15, random_state=0)\n",
+    "clf.fit(DX, DZ)\n",
+    "print(clf.feature_importances_)\n",
+    "scores = cross_val_score(clf, DX, DZ, cv=10)\n",
+    "print(scores)"
```
Comment (izgzhen) [inline classifier.ipynb]: What about putting three classifiers' scores in one dataframe and show it in notebook?

**Review by izgzhen — CHANGES_REQUESTED** [review]
Please see all the comments, thanks!

File: classifier.ipynb (hunk 2 above)
Comment (LuxiWang99) [reply classifier.ipynb]: We mentioned in the meeting, that instead of 10 numbers, we want 1 number. Just to clarify, the 1 number is the accuracy: the correct classification /  the total number of classification. So that each setup (ways of classification + hyperparameter) have a single accuracy.

File: data/graphs/backdoor/31a6a74c272af95f5223602e12ef1364527f584675445530eee060e635d2cafb.apk.top.dot (hunk 1 above)
Comment (LuxiWang99) [reply data/graphs/backdoor/31a6a74c272af95f5223602e12ef1364527f584675445530eee060e635d2cafb.apk.top.dot]: this is moved for convenience of testing, I will move it back

File: classifier.ipynb (hunk 2 above)
Comment (izgzhen) [reply classifier.ipynb]: Thanks for reminding me this - if so, should `score` instead of `scores` a better name. Also, please include the output of the running inside the notebook as well - this is why notebook is so useful to us :)

Comment (LuxiWang99) [reply classifier.ipynb]: Screen Shot 2019-07-28 at 7 00 24 PM

Comment (LuxiWang99) [reply classifier.ipynb]: The above is obtained by cross_val_predict and accuracy score.

**Review by izgzhen — APPROVED** [review]
Let's merge it - you might want to generalize and work on top of this in the next step
